regression.py

```diff
@@ -705,6 +706,9 @@ class GAMESSUSIRTest_ts(GenericIRTest):
     @unittest.skip('This is a transition state with different intensities')
     def testirintens(self):
         """This is a transition state with different intensities."""
+    @unittest.skip('This is a transition state with different atomcoords')

```

Comment:
What's the problem with this test? I can't for the life of me find the file(s) it runs for.